src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Commands/Cart/RemoveItemFromCart.xml

```diff
@@ -17,7 +17,7 @@
 >
     <class name="Sylius\Bundle\ApiBundle\Command\Cart\RemoveItemFromCart">
         <attribute name="orderItemId">
-            <group>shop:cart:remove_item</group>
+            <group>shop:order:remove_item</group>
         </attribute>
     </class>
```

Comment:
I'm wondering if this file is even needed. Delete endpoint does not have any body
![image](https://user-images.githubusercontent.com/6213903/144622042-029f66d4-33ff-4ffa-bc6b-974a2de16405.png)